### ISSUE TITLE
CLI executable error `/usr/local/bin/src: line 1: Not: command not found`

### ISSUE THREAD

**Issue by smaifullerton-wk** [sourcegraph/src-cli#219]
I'm seeing an error in Docker and when I install the sourcegraph CLI locally using the README instructions. This was not happening around 1-2 weeks ago. Seems like the download is not correct:

```
➜  ~ sudo curl -L https://sourcegraph.com/.api/src-cli/src_darwin_amd64 -o /usr/local/bin/src
Password:
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0      6      0  0:00:01  0:00:01 --:--:--     6
➜  ~ sudo chmod +x /usr/local/bin/src
➜  ~ src
/usr/local/bin/src: line 1: Not: command not found
➜  ~ src --help
/usr/local/bin/src: line 1: Not: command not found
```

This is the log from a successful curl when it was working:
```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                           Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   628  100   628    0     0    682      0 --:--:-- --:--:-- --:--:--   682
100 12.9M  100 12.9M    0     0  11.4M      0  0:00:01  0:00:01 --:--:-- 11.4M
```

**Comment by mrnugget** [comment]
Hey @smaifullerton-wk! Thanks for reporting this!

We made some small mistakes when releasing `3.14` and `3.15`. We didn't specify the full version name when tagging the release, we should've used `3.14.0` and `3.15.0`.

The result is that the forward from sourcegraph.com to the GitHub release doesn't work, becaus the tag doesn't exist.

We fixed that and are currently rolling out an update to Sourcegraph.com that should fix the download.
